include/common.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 #include <cctype>
 #include <string>
+#include <cstdint>
```

Comment:
```suggestion
#include <cstdint>
#include <string>
```

Includes should be ordered alphabetically.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
You don't need to fix this, as this repo now ignores the rule. (#18)